scripts/clean-all-caches.sh

```
@@ -0,0 +1,76 @@
+
+echo "📂 Project root folder: $PROJECT_ROOT"
+
+# Stop all processes
+echo "⏹️  Stopping processes..."
+pkill -f metro 2>/dev/null || true
+pkill -f node 2>/dev/null || true
+pkill -f gradle 2>/dev/null || true
+
+# Clean example project caches
+echo "🗂️  Cleaning example project caches..."
+cd "$PROJECT_ROOT/example"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)